core/templates/dev/head/pages/footer.html

```diff
@@ -5,7 +5,7 @@
         <h4 translate="I18N_FOOTER_ABOUT_ALL_CAPS"></h4>
         <ul>
           <li><a href="/about" translate="I18N_FOOTER_ABOUT"></a></li>
-          <li><a href="/about#foundation" translate="I18N_FOOTER_OPPIA_FOUNDATION"></a></li>
+          <li><a href="/foundation" translate="I18N_FOOTER_OPPIA_FOUNDATION"></a></li>
```

Comment:
Hm, just wondering, what are you planning to do about the foundation tab on the About page?

Reply:
For now, I'm thinking we should leave it. It links out to the Foundation site, but it shares information that's pertinent specifically to the users of Oppia.org -- namely, the license information and a shorter summary of what the organization is about without having to leave the site.

I think the site's informational pages as a whole could benefit from a broader audit, but for now leaving `/about#foundation` seems fine.